src/main/resources/xslt/camel/test-pom.xsl

```diff
@@ -112,12 +112,25 @@
         </xsl:copy>
     </xsl:template>
 
-  <!-- Forward solr.trust-store to the tests via surefire plugin -->
+  <!-- Add system property variables needed in Solr & Kafka Oauth Native Integration test -->
```

Comment:
Is there a reason to do it this way instead of configuring  `systemProperties` under the corresponding `test` element?
For example:
````
<test>
    <artifact>org.apache.camel.quarkus:camel-quarkus-integration-test-kafka-oauth:${camel-quarkus-test-list.version}</artifact>
    <systemProperties>
        <javax.net.ssl.trustStore>${solr.trust-store}</javax.net.ssl.trustStore>
    </systemProperties>
</test>
````

Reply:
No this was my first attempt. Should I ? It should be easier. Yet, I don' know why it's still failing

Reply:
I did that in one of the previous builds but it failed too.
I guess I need to move out the file from target to have it running here.
I'll close this PR, and try to do changes for next release. And see how it goes

Reply:
Let me know if I can help with the test config.